Signal/src/ViewControllers/DebugUI/DebugUIContacts.m

```diff
@@ -1107,7 +1107,9 @@ + (NSString *)randomLastName
         @"Noel",
         @"Vang",
     ];
-    return values[arc4random_uniform(values.count)];
+    
+    uint32_t index = arc4random_uniform((uint32_t)values.count);
```

Comment:
This actually still gives me a compiler warning:
 
<img width="579" alt="screen shot 2017-07-08 at 5 27 07 pm" src="https://user-images.githubusercontent.com/217057/27989640-b6fd27de-6402-11e7-9cd1-b94d8b5fadcd.png">